network.go

```diff
@@ -53,7 +60,7 @@ func initNetwork() error {
 		},
 		OnConfigChange: func(networkConfig *network.NetworkConfig) {
 			config.NetworkConfig = networkConfig
-			networkStateChanged()
+			networkStateChanged(false)
```

Comment:
If we're just reconfiguring things, likely the network isn't fully up yet, so we can wait for other events to fire.